dbt/models/sale/schema.yml

```diff
@@ -26,11 +27,11 @@ sources:
           - name: sv_is_ptax_outlier
             description: '{{ doc("shared_column_sv_is_ptax_outlier") }}'
           - name: sv_outlier_reason1
-            description: '{{ doc("shared_column_sv_outlier_reason") }}'
+            description: '{{ doc("shared_column_sv_outlier_reason1") }}'
           - name: sv_outlier_reason2
-            description: '{{ doc("shared_column_sv_outlier_reason") }}'
+            description: '{{ doc("shared_column_sv_outlier_reason2") }}'
           - name: sv_outlier_reason3
-            description: '{{ doc("shared_column_sv_outlier_reason") }}'
+            description: '{{ doc("shared_column_sv_outlier_reason3") }}'
```

Comment:
This was actually outdated

Reply:
**[Nitpick, blocking]** One small tweak here: Since the three outlier columns all share the same text for their descriptions, it actually fits our established style patterns for dbt docs to share a single description for them! So we can revert this change and just update the description for `shared_column_sv_outlier_reason` below.

```yaml
- name: sv_outlier_reason1
  description: '{{ doc("shared_column_sv_outlier_reason") }}'
- name: sv_outlier_reason2
  description: '{{ doc("shared_column_sv_outlier_reason") }}'
- name: sv_outlier_reason2
  description: '{{ doc("shared_column_sv_outlier_reason") }}'
```